test_plumbum.py

```diff
@@ -44,11 +44,11 @@ def test_no_template(self, mock_exit):
         we get the correct failure/exit.
         """
         args = [
-            'ec2',
             '-f', 'instance-type=c3.large',
+            'foo.yaml.j2',
```

Comment:
This completely broke this test. It may be passing, but it is not testing what this test is for. This is a test to make sure the template is required and `interpret_options()` fails. This is changed so that the template is now passed.